core/src/com/unciv/logic/automation/UnitAutomation.kt

```diff
@@ -270,6 +280,17 @@ class UnitAutomation{
         return false
     }
 
+    fun tryBombardEnemy(city: CityInfo): Boolean {
```

Comment:
If city can't attack, then we don't need to try to find an attackable enemy, this check should be first

Reply:
This is just a sanity check. Currently every turn each enemy city will run tryBombardEnemy() once, and is expected to be able to attack. Maybe delete this check?

Reply:
No, sanity checks are good